### ISSUE TITLE
object 'is_abs_path' not found

### ISSUE THREAD

**Issue by mquinson** [rstudio/bookdown-demo#67]
Hello, 

when I run `Rscript -e "bookdown::render_book('index.Rmd', 'bookdown::gitbook')"`, I get the following error message:
```
output file: bookdown-demo.knit.md

/usr/bin/pandoc +RTS -K512m -RTS bookdown-demo.utf8.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output bookdown-demo.html --lua-filter /usr/lib/R/site-library/bookdown/rmarkdown/lua/custom-environment.lua --lua-filter /usr/lib/R/site-library/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /usr/lib/R/site-library/rmarkdown/rmarkdown/lua/latex-div.lua --metadata-file /tmp/RtmpeK0v3x/filedc634e3f7795 --wrap preserve --standalone --section-divs --table-of-contents --toc-depth 3 --template /usr/lib/R/site-library/bookdown/templates/gitbook.html --highlight-style pygments --number-sections --css style.css --include-in-header /tmp/RtmpeK0v3x/rmarkdown-strdc63614fa11d.html --mathjax --filter /usr/bin/pandoc-citeproc 
Error in move_files_html(output, lib_dir) : 
  object 'is_abs_path' not found
Calls: <Anonymous> ... render_cur_session -> <Anonymous> -> <Anonymous> -> move_files_html
Execution halted
```

I tried to google a bit on this error message, but failed to find any hint. 

My system is a Debian testing and I installed the relevant packages from the Debian repositories. I can give you more details if needed, but so far I'm not even sure of what information could help you. Sorry about that.

**Comment by cderv** [comment]
Which versions of packages are you using ? 

If you update to last CRAN version of **knitr** and **bookdown**, it should be ok. This was an issue between package updates last year but shouldn't be an issue anymore. 

You should be using a recent **knitr** with an older **bookdown** problably. 🤔

**Comment by yihui** [comment]
Yes, that's the reason: your bookdown is way too old (from 2020), but knitr is relatively new. Personally I don't really recommend installing R packages from Debian repositories because you don't have control over when they can be updated. Especially for packages that do not require compilation, I'd just use `install.packages()` in R to install them. That way, you can easily update them via `update.packages()` at any time.

**Comment by mquinson** [comment]
For the record, I reported this bug against Debian here: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=991566

**Comment by yihui** [comment]
Thank you!
